extensions/amp-fit-text/1.0/fit-text.type.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** @externs */
+
+/**
+ * @typedef {{
+ *   minFontSize: number,
```

Comment:
Do we need to include `=` to specify that the font size props are optional?

Reply:
Can't do `=` for typedefs, unfortunately. But it's good idea to include `|undefined` for optional fields. I added them now. Good catch!

Reply:
Unfortunately `|undefined` fails with some Closure bug. It misses the fact that the default value is erases `|undefined` part and complains downstream that some calls are failing b/c `number|undefined` is not expected. Looking into this.